linux_os/guide/system/accounts/accounts-restrictions/var_accounts_authorized_local_users_regex.var

```diff
@@ -22,5 +22,6 @@ operator: pattern match
 interactive: true
 
 options:
+    rhel8: "^(root|bin|daemon|adm|lp|sync|shutdown|halt|mail|operator|games|ftp|nobody|pegasus|systemd-bus-proxy|systemd-network|dbus|polkitd|abrt|unbound|tss|libstoragemgmt|rpc|colord|usbmuxd$|pcp|saslauth|geoclue|setroubleshoot|rtkit|chrony|qemu|radvd|rpcuser|nfsnobody|pulse|gdm|gnome-initial-setup|postfix|avahi|ntp|sshd|tcpdump|oprofile|uuidd)$"
```

Comment:
On a vanilla RHEL8 I'm getting fails and the following users are being reported. I tested this with an empty `bad_user.fail.sh` file. I don't know if this should be reported as fail but 
I think it's worth investigating... maybe some account is not being defined in the regex.
```
/etc/passwd | daemon:
/etc/passwd | bin:
/etc/passwd | adm:
/etc/passwd | lp:
/etc/passwd | sync:
/etc/passwd | shutdown:
/etc/passwd | halt:
/etc/passwd | mail:
/etc/passwd | operator:
/etc/passwd | games:
/etc/passwd | ftp:
/etc/passwd | nobody:
/etc/passwd | dbus:
/etc/passwd | systemd-coredump:
/etc/passwd | systemd-resolve:
/etc/passwd | tss:
/etc/passwd | polkitd:
/etc/passwd | unbound:
/etc/passwd | sssd:
/etc/passwd | chrony:
/etc/passwd | sshd:
```